exercises/protein-translation/example.scala

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2017 © Denmar Technical Services Inc
+ * The U.S. Government has unlimited rights per DFAR 252.227-7014, all other
+ * rights reserved.
+ * <p>
+ * WARNING - This software contains Technical Data whose export is restricted by
+ * the Arms Export Control Act (Title 22, U.S.C., Sec 2751, et seq.) or the
+ * Export Administration Act of 1979, as amended (Title 50, U.S. C. App. 2401
+ * et seq.). Violations of these export laws are subject to severe criminal
+ * penalties.
+ */
+
+object ProteinTranslation {
+
+  private val translations = Map(
+    "AUG" -> "Methionine",
+    "UUU" -> "Phenylalanine",
+    "UUC" -> "Phenylalanine",
+    "UUA" -> "Leucine",
+    "UUG" -> "Leucine",
+    "UCU" -> "Serine",
+    "UCC" -> "Serine",
+    "UCA" -> "Serine",
+    "UCG" -> "Serine",
+    "UAU" -> "Tyrosine",
+    "UAC" -> "Tyrosine",
+    "UGU" -> "Cysteine",
+    "UGC" -> "Cysteine",
+    "UGG" -> "Tryptophan",
+    "UAA" -> "STOP",
+    "UAG" -> "STOP",
+    "UGA" -> "STOP")
+
+  def translate(input: String): Option[Seq[String]] = {
```

Comment:
I'm also in favor of keeping it as simple as possible, although I believe there are some languages that also use an option.